getSimulationSCSH.m

```diff
@@ -36,9 +36,11 @@
             case 'log'
                 tmp = arrayfun(@(x) diag(squeeze(SP.Cy(x,:,:))), 1:size(SP.Cy,1),'UniformOutput',false);
                 tmp = transpose([tmp{:}])/2;
-                my = [my; exp(SP.my + tmp)];
+                my_new = exp(SP.my + tmp);
+                my = [my; my_new];
```

Comment:
This `my_new`-thing is code-wise a nice idea. Thanks! :+1:

Reply:
Okay, hence also this is correct.
At least, it is for a diagonal covariance matrix... It might be interesting to know/check, whether this also holds true for a non-diagonal one... This I cannot guarantee right now...